### ISSUE TITLE
IPTorrents still not working on 0.7.885

### ISSUE THREAD

**Issue by Lusiphur** [Jackett/Jackett#951]
lusiphur@Tenth:/opt/Jackett$ mono JackettConsole.exe -v
Jackett v0.7.885.0

Test fails with ..

Test failed for iptorrents: Found no results while trying to browse this tracker

Logs

Jan 17, 2017 06:50:00 PM	Error	
Exception in test_indexer
Jan 17, 2017 06:50:00 PM	Info	
Found 0 releases from IPTorrents
Jan 17, 2017 06:49:54 PM	Info	
Manual search for "gotham" on IPTorrents with 0 results.
Jan 17, 2017 06:49:45 PM	Info	
Manual search for "Lucifer" on IPTorrents with 0 results.
Jan 17, 2017 06:49:38 PM	Info	
Manual search for "Lusiphur" on IPTorrents with 0 results.
Jan 17, 2017 06:49:06 PM	Error	
Exception in test_indexer
Jan 17, 2017 06:49:06 PM	Info	
Found 0 releases from IPTorrents
Jan 17, 2017 06:44:29 PM	Error	
Exception in test_indexer

**Comment by kaso17** [comment]
working fine for me, please run jackett with the -t option enabled and post the log

**Comment by Lusiphur** [comment]
01-17 19:02:12 Info Tracing enabled. 
01-17 19:02:13 Debug App config/log directory: /home/lusiphur/.config/Jackett 
01-17 19:02:13 Info Starting Jackett 0.7.885.0 
01-17 19:02:13 Info Using HTTP Client: UnixLibCurlWebClient 
01-17 19:02:13 Info Loading Cardigann definitions from: /home/lusiphur/.config/cardigann/definitions/ 
01-17 19:02:13 Info Loading Cardigann definitions from: /etc/xdg/cardigan/definitions/ 
01-17 19:02:13 Info Loading Cardigann definitions from: /opt/Jackett/Definitions 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/alphareign.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/apollo.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/bithq.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/bithumen.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/bitspyder.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/blubits.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/chdbits.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/datascene.yml 
01-17 19:02:13 Info Loading Cardigann definition /opt/Jackett/Definitions/eotforum.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/ethor.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/fanoin.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/freedomhd.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/funkytorrents.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/gormogon.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/hdclub.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/hdme.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/hdsky.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/hyperay.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/icetorrent.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/infinityt.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/mteamtp.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/myspleen.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/nachtwerk.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/nethd.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/newretro.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/qctorrent.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/rapidetracker.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/rockhardlossless.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/sdbits.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/secretcinema.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/shareisland.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/sharespacedb.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/shellife.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/tenyardtracker.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/thehorrorcharnel.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/theshinning.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/torrentbd.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/torrenting.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/torrentsectorcrew.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/torrentsmd.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/torviet.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/totheglory.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/trancetraffic.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/uhdbits.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/utorrents.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/worldofp2p.yml 
01-17 19:02:14 Info Loading Cardigann definition /opt/Jackett/Definitions/xtremezone.yml 
01-17 19:02:14 Info LibCurl init Ok 
01-17 19:02:14 Info LibCurl version libcurl/7.47.0 OpenSSL/1.0.2g zlib/1.2.8 libidn/1.32 librtmp/2.3 
01-17 19:02:14 Info Starting web server at http://*:9117/ 
01-17 19:02:15 Error Top level exception System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Net.Sockets.SocketException: Address already in use
  at System.Net.Sockets.Socket.Bind (System.Net.EndPoint local_end) <0x413f3f90 + 0x000ef> in <filename unknown>:0 
  at System.Net.EndPointListener..ctor (System.Net.IPAddress addr, Int32 port, Boolean secure) <0x413f1490 + 0x00147> in <filename unknown>:0 
  at System.Net.EndPointManager.GetEPListener (System.String host, Int32 port, System.Net.HttpListener listener, Boolean secure) <0x413f0ff0 + 0x002d3> in <filename unknown>:0 
  at System.Net.EndPointManager.AddPrefixInternal (System.String p, System.Net.HttpListener listener) <0x413ef3a0 + 0x000c3> in <filename unknown>:0 
  at System.Net.EndPointManager.AddListener (System.Net.HttpListener listener) <0x413eeeb0 + 0x00147> in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x7ff5bddab9e0 + 0x000ef> in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) <0x7ff5bdbe3cf0 + 0x0002a> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.ServerFactory.ServerFactoryAdapter.Create (IAppBuilder builder) <0x413eb140 + 0x001f2> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Engine.HostingEngine.StartServer (Microsoft.Owin.Hosting.Engine.StartContext context) <0x413eb100 + 0x00029> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Engine.HostingEngine.Start (Microsoft.Owin.Hosting.Engine.StartContext context) <0x413be8e0 + 0x0015b> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Starter.DirectHostingStarter.Start (Microsoft.Owin.Hosting.StartOptions options) <0x413bdc20 + 0x0005c> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Starter.HostingStarter.Start (Microsoft.Owin.Hosting.StartOptions options) <0x413bc740 + 0x0007b> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.StartImplementation (IServiceProvider services, Microsoft.Owin.Hosting.StartOptions options) <0x413bc110 + 0x0005e> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.Start (Microsoft.Owin.Hosting.StartOptions options) <0x413badc0 + 0x00027> in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.Start[TStartup] (Microsoft.Owin.Hosting.StartOptions options) <0x413bacd0 + 0x00047> in <filename unknown>:0 
  at Jackett.Services.ServerService.Start () <0x413ba220 + 0x0020b> in <filename unknown>:0 
  at JackettConsole.Program.Main (System.String[] args) <0x41251d60 + 0x00e16> in <filename unknown>:0

**Comment by Ryanevans94** [comment]
I've got the same issue running Jackett Version 0.7.876.0 was working yesterday must be a change IPT have made.

**Comment by kaso17** [comment]
You're trying to start a second instance of jackett on the same port:
System.Net.Sockets.SocketException: Address already in use
Please stop the already running one first.

**Comment by ZoeyCluff** [comment]
I copy pasted my cookie from sickbeard and it's saying it's invalid.

**Comment by kaso17** [comment]
please enable enhanced logging and post the log

**Comment by Lusiphur** [comment]
I appear to only have one instance running.

lusiphur@Tenth:/opt/Jackett$ ps -ef | grep -i mono
lusiphur   825     1  6 17:44 ?        00:00:07 /usr/bin/mono --debug /opt/NzbDrone/NzbDrone.exe -nobrowser
lusiphur   831     1  1 17:44 ?        00:00:01 /usr/bin/mono --debug /opt/jackett/JackettConsole.exe
lusiphur  1820  1748  0 17:46 pts/0    00:00:00 grep --color=auto -i mono

I tried the following. 
1) Stop the jackett service. Restart it. Same issue.
2) Stop the jackett service. Reboot. Same issue.

**Comment by kaso17** [comment]
Please check if there's another process using the port.
netstat -lnp | grep 9117
otherwise just enable enhanced logging in the webinterface and post the log.txt file..

**Comment by Lusiphur** [comment]
lusiphur@Tenth:/opt$ sudo netstat -lnp | grep 9117
tcp        0      0 0.0.0.0:9117            0.0.0.0:*               LISTEN      12577/mono      
[log.20161019.00000.txt](https://github.com/Jackett/Jackett/files/715070/log.20161019.00000.txt)

**Comment by Lusiphur** [comment]
And I have just noticed that jackett isn't creating a log file in its directory. The one I posted (which is the only one there) hasn't been updated since 19th October 2016.

**Comment by Lusiphur** [comment]
Managed to get Jackett working.
With no logging at all I thought the best thing to do was re-install. So I,
1) stopped the service
2) deleted /opt/jackett
3) Downloaded the most recent version and re-installed according to the instructions

I could not configure a new connection to IPTorrents due to it wanting a captcha. I followed the linked instructions to extract the cookie and it all works now.

I would close this but there seems to be other people getting their issues resolved on this thread.
